packages/ng/styles/components/_dropdown.scss

```diff
@@ -1,15 +1,14 @@
 :root {
-	--components-dropdown-width: 14rem;
-	--components-dropdown-max-width: 90vw;
+	--components-dropdown-min-width: 10rem;
```

Comment:
why fix a min-width instead of take the content width ?

Reply:
It takes the content width. It's just a way to avoid very narrow dropdowns.